install_scripts/templates/kubernetes-init.sh

```diff
@@ -247,6 +248,12 @@ rookDeploy() {
 }
 
 contourDeploy() {
+    # DISABLE_CONTOUR
+    if [ -n "$1" ]; then
+        printf "${YELLOW}contour ingress controller not deployed\n"
```

Comment:
I don't think this should be yellow. May not even need to be printed. Yellow is for warnings.